### ISSUE TITLE
Deploying breaks after a while: CheckNetIsolation LoopbackExempt -a -n=

### ISSUE THREAD

**Issue by Maniacsh** [Microsoft/react-native-windows#1042]
I am in the very babysteps of making my UWP-app with React Native.

Somehow I never get to successfully deploy the (Hello World) app to an emulator.

Deploying on my laptop itself does succeed however.

When deployed to the emulator I get an error telling me that DLL for Yoga could not be loaded.

After that when I try to deploy again on my local system (laptop), this error occurs:

`CheckNetIsolation LoopbackExempt -a -n=`

Now all I am able to do is run `react-native windows` to start all over again.

Am I doing something wrong? Or is there something wrong with the Yoga DLL?

**Comment by rozele** [comment]
@maanishIE This, unfortunately, is a known issue with react-native-windows.  I think somehow the loopback exemption table gets corrupted when you deploy the app from Visual Studio.

The deployment from Visual Studio should not give you any issues. The reason running react-native windows fixes the issue is because it generates a new GUID for your app ID. 

It may be the case that we can just ignore the LoopbackExempt warning (if a record already exists for the app anyway), if you can reproduce the issue, can you comment out this line:

https://github.com/ReactWindows/react-native-windows/blob/master/local-cli/runWindows/utils/deploy.js#L99

And see if react-native run-windows works against the packager server? Note, you may have to Ctrl+R to reload the JavaScript from the packager server because it may try to use the cached bundle.  If it works, we can wrap a try catch around that.

**Comment by Foovanadil** [comment]
@rozele just and FYI, I just started getting this issue locally and found this github issue thread. I tried commenting out the line you indicated and tried running react-native run-windows again and received the following error:


`Removing old version of the app
Installing new version of the app
Starting the app
Get-AppxPackageManifest : Cannot process argument transformation on parameter 'Package'. Object reference not set to an instance of an object.
At C:\MyProject\node_modules\react-native-windows\local-cli\runWindows\utils\WindowsStoreAppUtils.ps1:160 char:41
+     $manifest = Get-appxpackagemanifest $package
+                                         ~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Get-AppxPackageManifest], ParameterBindingArgumentTransformationException
    + FullyQualifiedErrorId : ParameterArgumentTransformationError,Microsoft.Windows.Appx.PackageManager.Commands.GetAppxPackageManifestCommand

Exception calling "ActivateApplication" with "4" argument(s): "Value does not fall within the expected range."
At C:\MyProject\node_modules\react-native-windows\local-cli\runWindows\utils\WindowsStoreAppUtils.ps1:168 char:5
+     $appActivator.ActivateApplication($applicationUserModelId,$args,[ ...
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : ArgumentException
`

**Comment by Foovanadil** [comment]
@rozele also, running react-native windows didn't seem to fix the problem for me.

I ran react-native windows. which regenerated the windows SLN and code and then I ran react-native run-windows and still received the same error about CheckNetIsolation failing...


So I am currently dead in the water..

**Comment by rachmcnamara** [comment]
I also had this same issue - from the Windows Control Panel, I "repaired" my VS installation and this solved the problem.  Note however that this takes like 30-40 minutes to complete - so go get a coffee!

**Comment by ayunawani** [comment]
Hi, 
I am also facing this same issue. I am not able to run  from cli  using command `react-native run-windows.`
And getting the same errror `CheckNetIsolation LoopbackExempt -a -n=`
But I am able to Run Project from Visual Studio.
Any idea how can i run from cli?

**Comment by pranavrajsinghchauhan** [comment]
I too facing the same issue, any help would be very much appreciated

**Comment by eltiare** [comment]
I got this error, but right before it I saw the error that the Add-AppDevPackage.ps1 was unable to be executed. When I try to run it in powershell it complains:

```& ".\windows\test_app\AppPackages\test_app_1.0.0.0_x86_Debug_Test\Add-AppDevPackage.ps1"
Found package: X:\test_app\windows\test_app\AppPackages\test_app_1.0.0.0_x86_Debug_Test\test_app_1.0.0.0_x86_Debug.appx
Error: The package or bundle is not digitally signed or its signature is corrupted.```

**Comment by eltiare** [comment]
I've disabled signature checking on Windows 10 Home, and now it is giving me this error: 

```& : The term 'windows/test_app/AppPackages/test_app_1.0.0.0_x86_Debug_Test/Add-AppDevPackage.ps1' is not recognized as
the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was
included, verify that the path is correct and try again.
At line:1 char:3
+ & "windows/test_app/AppPackages/test_app_1.0.0.0_x86_Debug_Test/Add-App ...
+   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (windows/test_app...pDevPackage.ps1:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

Failed to deploy: Command failed: CheckNetIsolation LoopbackExempt -a -n=```

**Comment by nowtech** [comment]
Anyone found a solution for this? 

Having same issue with clean Visual Studio 2015 Professional install and clean react-native project:
react-native init winbase
cd winbase
react-native i --save-dev rnpm-plugin-windows
react-native windows
react-native run-windows

**Comment by eltiare** [comment]
Ultimately my problem was caused by not having proper permissions on the folder. It was a network share with Parallels between Windows and OS X. I moved it to my documents folder and it worked fine. I tried building in VS 2015 to get a better error message, which led me to my solution.

**Comment by nowtech** [comment]
ok, will try that, however running on a standalone windows pc...

**Comment by amorenew** [comment]
I opened the project in Visual Studio 2015 and run it from there.

**Comment by jpshelley** [comment]
i get the same issue as @Foovanadil after commenting out that line @rozele 
```
Get-AppxPackageManifest : Cannot process argument transformation on parameter 'Package'. Object reference not set to an instance of an object
At C:\Users\hudl\Developer\courtside\node_modules\react-native-windows\local-cli\runWindows\utils\WindowsStoreAppUtils.ps1:160 char:41
+     $manifest = Get-appxpackagemanifest $package
+                                         ~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Get-AppxPackageManifest], ParameterBindingArgumentTransformationException
    + FullyQualifiedErrorId : ParameterArgumentTransformationError,Microsoft.Windows.Appx.PackageManager.Commands.GetAppxPackageManifestComma

Exception calling "ActivateApplication" with "4" argument(s): "Value does not fall within the expected range."
At C:\Users\hudl\Developer\courtside\node_modules\react-native-windows\local-cli\runWindows\utils\WindowsStoreAppUtils.ps1:168 char:5
+     $appActivator.ActivateApplication($applicationUserModelId,$args,[ ...
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : ArgumentException
```

**Comment by jpshelley** [comment]
I was able to get around this error finally:
```
1. rimraf .\node_modules\
2. npm cache verify
3. npm install
3. react-native run-windows --arch x64
```
For me the issue was that the run-windows command didn't add my cert to the personal user's cert store. We had 2 certs in our repo and I had to manually add both of them by double clicking them.

There is still one issue where the run-windows command won't launch the react packager, so I need to then run `react-native start` once the app has launched. I'll create a new ticket for this issue though.

**Comment by rozele** [comment]
Closing due to lack of activity. Generally i've encountered this when trying to execute code from a Mac OS directory through Parallels.

**Comment by nageshwar521** [comment]
> I was able to get around this error finally:
> 
> ```
> 1. rimraf .\node_modules\
> 2. npm cache verify
> 3. npm install
> 3. react-native run-windows --arch x64
> ```
> 
> For me the issue was that the run-windows command didn't add my cert to the personal user's cert store. We had 2 certs in our repo and I had to manually add both of them by double clicking them.
> 
> There is still one issue where the run-windows command won't launch the react packager, so I need to then run `react-native start` once the app has launched. I'll create a new ticket for this issue though.
> 
> edit: I think the last statement has been known and fixed in a more recent version #1250

`react-native run-windows --arch x64`

@jpshelley thank you, it solves the issue...you saved my time
